forms-flow-web/src/components/PrivateRoute.jsx

```diff
@@ -88,9 +88,10 @@ const PrivateRoute = React.memo((props) => {
   const viewTasks = userRoles.includes("view_tasks");
   const manageTasks = userRoles.includes("manage_tasks");
   const viewDashboards = userRoles.includes("view_dashbaords");
-
-
-
+  const isAdmin = userRoles.includes("admin");
+  const isCreateSubmissions = userRoles?.includes("create_submissions");
+  const isViewDesigns = userRoles?.includes("view_designs");
```

Comment:
do we need to check like this because we already have one function  in forms-flow-web\src\constants\permissions.js

Reply:
Updated